cmd/experimental/podtaintstolerations/controller/pod_jobs.go

```diff
@@ -0,0 +1,197 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"context"
+	"sort"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
+	"sigs.k8s.io/kueue/pkg/controller/jobframework"
+)
+
+var AdmissionTaintKey = "kueue.x-k8s.io/kueue-admission"
+
+const (
+  ControllerName = "kueue-podtaintstolerations"
+)
+
+var (
+	GVK = corev1.SchemeGroupVersion.WithKind("Pod")
+
+	FrameworkName = "core/pod"
+)
+
+var NewReconciler = jobframework.NewGenericReconciler(func() jobframework.GenericJob { return &Pod{} }, nil)
+
+type Pod corev1.Pod
+
+var (
+	_ jobframework.GenericJob           = (*Pod)(nil)
+	_ jobframework.JobWithCustomStop    = (*Pod)(nil)
+	_ jobframework.JobWithPriorityClass = (*Pod)(nil)
```

Comment:
I think you don't need to implement this one. The framework should get it from `PodSets()`.